README.md

```diff
@@ -4,13 +4,27 @@ This contains the source for QuPath's documentation, hosted at https://qupath.re
 
 ## Building locally
 
-To build this locally, you should first install (possibly in a [venv](https://docs.python.org/3/library/venv.html)):
-- `sphinx-build`
-- `sphinx_rtd_theme`
-- `myst_parser`
-- `readthedocs-sphinx-search`
+To build this locally, create a virtual environment, eg:
 
-As well as the command line tool `Make` (e.g., [GNU Make](https://www.gnu.org/software/make/)).
+```bash
+python -m venv ./venv
+. ./venv/bin/activate
+```
+
+or using conda/mamba:
+
+```bash
+conda env create -n qupath-docs
+conda activate qupath-docs
```

Comment:
You'll need to install Python here

Reply:
I thought conda envs generally come with python and pip?

Reply:
It seems not, unless you use
```
mamba create -n qupath-docs python
```
May be safer to specify version, e.g.
```
mamba create -n qupath-docs python=3.12
```